src/constants/rolechecker.js

```diff
@@ -0,0 +1,27 @@
+const bot = require('../bot')
+const masterID = '127060142935113728'
+
+bot.checkRole = async (msg, roleArr) => {
+  var returnobj;
```

Comment:
In JS, variables are almost always camel-case. So in the case of this one it should be
`var returnObj;`

Reply:
And again here, this is going to be a boolean so call it something like `hasRole`